### ISSUE TITLE
Feature Request: Option for users to edit other's recipies

### ISSUE THREAD

**Issue by Don-Swanson** [open-eats/OpenEats#97]
I would like to see a permission that you can grant to specific users (say super users, but not admins) the ability to edit any recipe (currently you can only edit your own recipes)

**Comment by putt1ck** [comment]
Even neater would be the option to fork the recipe. So you don't so much as edit the original as create a new one and then edit that - which is automatically shared to the group (and the originator notified, in case it's a fix more than a change, which they can then merge into their original).

**Comment by Don-Swanson** [comment]
I agree that that would be even neater. My idea though is less confusing for the general population that has no idea how github and the whole fork/merge process works. 

My use case is that I have an admin account that I usually add recipes with, and my wife has a not so admin account since she has no clue what she's doing and I don't want her to accidentally make some major backend changes without knowing.

**Comment by putt1ck** [comment]
Surely we can present fork and merge in non-tech language i.e. achieve both use cases with the same functionality. Like if you view a recipe the option is to make changes, which (if it's not your recipe) automatically becomes a fork, then if you view either it's flagged that a variation exists (with an option somewhere to add some notes to the variations).

**Comment by Don-Swanson** [comment]
To be honest, that just sounds way more complicated and cluttered. I also believe that simply giving permissions to edit other people's recipes is a simple backend change (I just don't know enough about the platform in use to do it myself).

What you are suggesting would require a whole lot of code and management. Now, don't get me wrong, I like your idea, but with my experience of the masses they either aren't going to know what to do, or be dangerous enough and end up with 20 different recipes of the same thing because they didn't want to add the optional almonds to their recipe. Plus if you wanted to go the whole fork/merge route, you could just build and host a website with github, and it already has your desired functionality built in.

**Comment by RyanNoelk** [comment]
The idea to grant an edit permissions on recipes is a good idea. I'll have some time in a couple months to add this.

**Comment by felinira** [comment]
This would be a really nice feature. Just started using it and would like to edit other people's recipes as well. Maybe you could just add users to a group where everyone can edit all their recipes? I would even be fine if every user could edit everything.

**Comment by davek79** [comment]
Yeah +1 on this one.  Once I got past the confusion of setup, this is exactly what I'm looking for.  Thanks for writing this!
